pkg/transport/transport.go

```diff
@@ -37,6 +37,9 @@ type Transport struct {
 	requestTimeout time.Duration
 }
 
+// TCP represents TCP network type
+const TCP = "tcp"
```

Comment:
I'm not certain that this improves much, even more I'm more accustomed to seeing "tcp" as an argument rather than a constant.

Reply:
yeah, I was too fast with this